server/ai_live_video.go

```diff
@@ -409,7 +466,10 @@ func startControlPublish(ctx context.Context, control *url.URL, params aiRequest
 				}
 				// if there was another type of error, we'll just retry anyway
 			case <-done:
+				stopProcessing(ctx, params, fmt.Errorf("stopping control publisher"))
```

Comment:
Is this necessary here - what scenarios do we get a `<-done` trigger without the orch being alive? Would like to avoid sprinkling cleanup code around without fully understanding the flow control